homework-submission/homework w1/app.js

```diff
@@ -0,0 +1,114 @@
+const ListItem = ({ description, deadLine, done }) => {
+  return <li className={done}>{description + ', ' + deadLine}</li>;
+};
+
+const Button = ({ text, type, event }) => {
```

Comment:
a better name would be for the the prop would be `clickHandler` or simply `onClick`.
as this function **onClick** takes an *event* and does something with it.
so onClick would be something like
```
onClick={event => doSomethingWith(event)}
```

Reply:
ok I think so I'll use it from now on